### PR TITLE
Changed var to let/const, added es6 to env and matching rules

### PR DESCRIPTION
**Pull request by atomtigerzoo**
This should close issue #32.

I added the matching es6 rule to the environment and also the two rules for const/let linting. I hope you like the changes I made.

### CODE REVIEW

**Comment by butlerx** [comment]
hey you seem to be getting errors with the minification

**Comment by atomtigerzoo** [comment]
The build is passing now. I had to add babel to the packages to compile the ES5/6 to standard javascript so that the minification can kick in. Without babel the minification will fail.

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015"],
```
Comment (butlerx) [inline .babelrc]: Could you move this in to the package.json

Comment (atomtigerzoo) [inline .babelrc]: I moved both settings, if that is ok with you, and updated the PR.

**Pull request merged**